api/dps/index_internal_test.go

```diff
@@ -336,21 +335,21 @@ func TestIndex_Height(t *testing.T) {
 		index := Index{
 			client: &apiMock{
 				GetHeightForBlockFunc: func(_ context.Context, in *GetHeightForBlockRequest, _ ...grpc.CallOption) (*GetHeightForBlockResponse, error) {
-					assert.Equal(t, mocks.ByteSlice(mocks.GenericIdentifier(0)), in.BlockID)
+					assert.Equal(t, mocks.ByteSlice(blockID), in.BlockID)
 
 					return &GetHeightForBlockResponse{
-						BlockID: mocks.ByteSlice(mocks.GenericIdentifier(0)),
+						BlockID: mocks.ByteSlice(blockID),
 						Height:  mocks.GenericHeight,
 					}, nil
 				},
 			},
 		}
 
-		got, err := index.HeightForBlock(mocks.GenericIdentifier(0))
+		got, err := index.HeightForBlock(blockID)
```

Comment:
In the end it's now

```go
		got, err := index.HeightForBlock(blockID)

		require.NoError(t, err)
		assert.Equal(t, header.Height, got)
```

Is that satisfactory for you or did you prefer with `want`/`got`?

I'm also fine with renaming `got` in `height` and such, since there's no want anymore, it might be a bit weird to have _gots_ lying around. WDYT?

Reply:
I'm OK leaving it like this.